SETUP.md

```diff
@@ -100,7 +100,7 @@ You'll want to downlaod and install PostgreSQL onto your system through their [s
 
 Launch the PostgreSQL manager (I do it though the `psql` command in my terminal) and create the database by:
 
-    create database stem_ecosystem;
+    CREATE database stem_ecosystem;
```

Comment:
Did you run into issues with this step? All caps is a SQL convention preference but it shouldn't throw errors anywhere.

Reply:
In windows, it worked only when I used uppercase letters and so I changed it.